meisel/bridge.h

```diff
@@ -0,0 +1 @@
+#import "header.h"
```

Comment:
ah, I think since this is using quote imports, you might also need to generate an hmap for the target being imported, or else point at the `MyApp_private_hmap` manually